bridge_adaptivity/module/engines/engine_vpal.py

```diff
@@ -134,7 +134,7 @@ def select_activity(self, sequence):
         reco_url = urllib.parse.urljoin(
             "{}/".format(self.activity_url), "recommend"
         )
-        payload = {"collection": sequence.collection.slug, "sequence": []}
```

Comment:
it looks like a bad idea to exchange slug with the pk.

Reply:
I fixed it.